bee-runtime/CHANGELOG.md

```diff
@@ -0,0 +1,31 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+<!-- ## Unreleased - YYYY-MM-DD
+
+### Added
+
+### Changed
+
+### Deprecated
+
+### Removed
+
+### Fixed
+
+### Security -->
+
+
+## 0.1.0-alpha - 2021-01-08
+
+### Added
+
+- `Node` and `NodeBuilder` traits
+- `Worker` trait
+- `ResHandle` resource handle
```

Comment:
If you change the name, don't forget it here

Reply:
Already done :)